vitest.config.js

```diff
@@ -0,0 +1,31 @@
+import { defineConfig, mergeConfig } from 'vitest/config'
+import viteConfig from './vite.config'
+
+export default defineConfig(configEnv => mergeConfig(
+  viteConfig(configEnv),
+  defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    alias: {
+      "\\.css$": "identity-obj-proxy",
+      "\\.svg\\?react$": "<rootDir>/test/mocks/svgr.ts",
+      "^\\./IndexedDBWorker\\?worker$": "<rootDir>/test/mocks/workerMock.ts",
+      "^\\./olm$": "<rootDir>/test/mocks/olmMock.ts"
+    },
```

Comment:
Part of the advantage of using vitest is that because it shares configuration with Vite, you don't have to mock out quite as many things by hand. I believe we can safely remove this entire `alias` block.

Reply:
Done!